### PR TITLE
chore(build): remove ovh-ui-kit from main file

### PR DESCRIPTION
**Pull request by AxelPeter**
The goal of this version is to remove `ovh-ui-kit` from the main file to avoid version conflicts.

* Remove `ovh-ui-kit` from the main CSS file
* Create a Makefile
* Add new NPM scripts
* Move all fonts files in `./assets`
* Move files for test page in `./docs`

**With this minor version, we need to import `ovh-ui-kit` in our project's LESS file.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,43 +16,42 @@ npm install --save-dev ovh-ui-kit-bs
...
-<link rel="stylesheet" href="./node_modules/ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css">
+<link rel="stylesheet" href="./node_modules/ovh-ui-kit-bs/dist/ovh-ui-kit-bs.min.css">
 ```
 
 #### less
 
-[less-plugin-remcalc](https://github.com/ovh-ux/less-plugin-remcalc) is needed to compile the LESS import.
+Those dependencies are needed to compile the LESS import.
+
+- [bootstrap](https://github.com/twbs/bootstrap)
```
Comment (antleblanc) [inline README.md]: capital **B** please :)

File: package.json [hunk 2]
```diff
@@ -14,33 +14,33 @@
   },
   "scripts": {
     "build": "npm run clean && npm run less && npm run copy",
-    "clean": "rimraf dist",
+    "clean": "rimraf dist && rimraf docs && rimraf assets",
```
Comment (antleblanc) [inline package.json]: or `rimraf dist docs assets`

File: README.md (hunk 1 above)
Comment (AxelPeter) [reply README.md]: For `Bootstrap` ? I don't know...
It was to keep a coherence by using dependencies name for all :)

File: package.json (hunk 2 above)
Comment (AxelPeter) [reply package.json]: Didn't know. Done :)

**Comment by AxelPeter** [comment]
Added

**Pull request merged**